README.md

```diff
@@ -15,14 +15,19 @@ $ jx add app jx-app-jacoco --repository "http://chartmuseum.jenkins-x.io"
 
 NOTE: The syntax of this command is evolving and will change.
 
-Upon successful installation, you should see jx-app-jacoco in the list of pods (`kubectl get pods`) running in your cluster - it will be called `jx-app-jacoco-jx-app-jacoco`.
+After the installation, you can view the status of jx-app-jacoco via:
+
+```bash
+$ helm status jx-app-jacoco
```

Comment:
Add a note here that this is a workaround until we implement `jx get apps`

Reply:
Done